bot/cogs/source.py

```diff
@@ -16,15 +16,23 @@ class Source(commands.Cog):
     def __init__(self, bot: commands.Bot) -> None:
         self.bot = bot
 
+    @in_channel(constants.Channels.bot)
     @commands.command(name="source")
-    async def command_source(self, ctx: commands.Context, command_name: str = None) -> None:
+    async def command_source(self, ctx: commands.Context, command_name: str = None, sub_command: str = None) -> None:
```

Comment:
At the current state, only one command and one subcommand are supported. Why couldn't we do this? : `async def command_source(self, ctx: commands.Context, *, command_name: str = None) -> None:`